src/Microsoft.Data.Sqlite/Utilities/NativeLibraryLoader.cs

```diff
@@ -103,6 +80,8 @@ public static bool IsLoaded(string dllName)
             return handle != IntPtr.Zero;
         }
 
+        private static bool IsDNX() => PlatformServices.Default != null;
```

Comment:
Hmm, it seems to work, but I'll change it to `PlatformServices.Default?.Application.RuntimeFramework.Identifier == "DNX"` just to be safe.